modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -61,10 +59,8 @@ logrotate /etc/logrotate.conf
 # Change the access level of configs/ rules/ and run_forseti.sh
 chmod -R ug+rwx ${forseti_home}/configs ${forseti_home}/rules ${forseti_home}/install/gcp/scripts/run_forseti.sh
 
-# Install tracing libraries
-pip install .[tracing]
-
 # Install Forseti
+echo "Installing Forseti"
```

Comment:
Does this change provide value? If not, can we omit it?

Reply:
It provided a value when looking through serial console output. If startup script ever fails i find this comment valuable in the console logs.

Reply:
In terms of functionality this change is negligible, so we can keep it in. However let's minimize changes like this in the future - if there's some subtlety that we need to draw attention let's add logging statements, but we should avoid leaving debug logging in pull requests.